functions/Restore-DbaDatabase.ps1

```diff
@@ -221,8 +221,30 @@ folder for those file types as defined on the target instance.
     }
     PROCESS
     {
-        
-        foreach ($f in $path)
+		$base = $SqlServer.Split("\")[0]
+		
+		if ($base -eq "." -or $base -eq "localhost" -or $base -eq $env:computername -or $base -eq "127.0.0.1")
+		{
+			# they can use whatever path they like
+		}
+		else
+		{
+			if ($Path.StartsWith("\\") -eq $false)
+			{
+				# Many internal functions parse using Get-ChildItem. 
+				# We need to use Test-SqlPath and other commands instead
+				# Prevent people from trying 
+				
+				Write-Warning "Currently, you can only use UNC paths when running this command remotely. We expect to support non-UNC paths for remote servers shortly."
+				return
+				
```

Comment:
Naw. When you call "continue" within a loop (such as the foreach-loop here), it will stop the current iteration and move to the next item. If it was the last item, it will gracefully leave the loop and continue with End {}